app.py

```diff
@@ -177,20 +166,17 @@ def delete_entry():
         Later I'll replace with a database algorith"""
 
         index = int(flask.request.form["Delete"])
-        # Later, I'll store the following algorithm in another file
-        entry = Entry.query.filter_by(id=index)
-        if entry:
-            db.session.delete(entry)
-            db.session.commit()
+        # The following algorithm in the database functions file
+        deleteEntry(index)
     return flask.redirect(flask.url_for("users_entries"))
 
 
 @app.route("/add_entry", methods=["GET", "POST"])
 def add():
     # new entry object information
     poster = current_user.id
-    title = flask.request.form("title")
-    contents = flask.request.form("entry")
+    title = flask.request.form["title"]
```

Comment:
I made this correction because, I needed for this to be in brackets not parenthesis.